### ISSUE TITLE
Incorrect text in log message

### ISSUE THREAD

**Issue by ToneD** [ToneD/grunt-niteo-awscloudformation#1]
lib/awscloudformation.litcoffee:270

grunt.log.ok "Stack #{@data.name} does not exist."

should be

grunt.log.ok "Stack #{@data.name} exists."

**Comment by mitchelldavis** [comment]
@ToneD, Hey man.  Just saw this, did you mean to add this to the upstream repo?

**Comment by ToneD** [comment]
@mitchelldavis i was going to include it with my next pull request, but if it's the type of thing that annoys you then it's just a quick update :)

**Comment by mitchelldavis** [comment]
No, that works.  I was just curious.  Thanks for everything @ToneD
